src/styles/blocks/card.scss

```
@@ -0,0 +1,98 @@
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  justify-content: space-between;
+  flex-direction: column;
+
+  background-color: $white-color;
+  border: 1px solid $border-color;
+  border-radius: 5px;
+
+  &__img {
+    margin: 0 auto 40px;
+    height: 134px;
+    width: 160px;
+  }
+
+  &__details {
+    display: flex;
+    justify-content: flex-start;
+    flex-direction: column;
+    margin-bottom: 16px;
+
+    line-height: 14px;
+
+    &-title {
+      margin-bottom: 4px;
+      color: $review-color;
+      line-height: 18px;
+      font-size: 12px;
+      font-weight: 500;
+    }
+
+    &-identifier {
+      font-size: 10px;
+      font-weight: 400;
+      align-items: center;
+      color: $info-color;
+    }
+  }
+
+  &__rating {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+
+    &-info {
+      font-size: 10px;
+      font-weight: 400;
+      line-height: 14px;
+      color: $review-color;
+    }
+  }
+
+  &__price {
+    display: flex;
+    justify-content: space-between;
+    margin-bottom: 16px;
+
+    &-sum {
+      font-size: 16px;
+      font-weight: 700;
+      color: $review-color;
+      line-height: 18px;
+    }
+
+    &-text {
+      color: $info-color;
+      font-size: 12px;
+      font-weight: 400;
+      line-height: 18px;
+    }
+  }
+
+  &__link {
+    display: block;
+    text-decoration: none;
+    width: 100%;
+    height: 40px;
+    text-align: center;
+    border: 1px solid;
```

u can also use color here
https://developer.mozilla.org/ru/docs/Web/CSS/border

```suggestion
    border: 1px solid $buy-button-color;
```